downloads/swan-lake-release-notes/swan-lake-beta3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,794 @@
+- Made the access level of the `WatchEvent` to `readonly` in the FTP listener
+
+##### `sql` Package
+- Improved the throughput performance with asynchronous database queries
+- Introduced new array out parameter types in call procedures
+- Changed the return type of the SQL query API to include the completion type as nil in the stream. The SQL query code below demonstrates this change
+    
+    **Previous Syntax**
+
+    ```ballerina
+    stream<RowType, error> resultStream = sqlClient->query(``);
+    ```
+
+    **New Syntax**
+    ```ballerina
+    stream<RowType, error?> resultStream = sqlClient->query(``);
+    ```
+
+- Improved the error types in the SQL module with the introduction of typed errors for data manipulation under the `sql:ApplicationError`.
+- Removed support for the string query parameter.
+
+    **Previous Syntax**
+
+    ```ballerina
+    stream<RowType, error?> resultStream = sqlClient->query("SELECT * FROM Students;");
+    ```
+
+    **New Syntax**
+    ```ballerina
+    stream<RowType, error?> resultStream = sqlClient->query(`SELECT * FROM Students;`);
+    ```
+
+##### `io` Package
+- Changed the `io:readin` function input parameter to optional. In the previous API, it was required to pass a value to be printed before reading the user input as a string. It was removed due to the breaking change and made optional. It is not recommended to pass a value to print it in the console.
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta3](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+Beta3%22+label%3AType%2FBug).
+
+### Code to Cloud Updates
+
+#### Improvements
+- Added a flag to disable the Docker image generation
+- Added the secret generation support for HTTP clients
+- Improved diagnostics for failed value retrievals
+- Added support for multiple volumes
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta3](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta3%22).
+
+### Dependency Management Updates
+
+With Swan Lake Beta3, the way how dependencies of a package are managed has been changed. The new implementation will ensure the following. 
+
+ - The new build will ensure that you will get updates of the dependent packages (direct or transitive) automatically
+ - Introducing the `--sticky` flag in case you want to lock the dependency versions for subsequent builds
+
+The following changes have been introduced.
+
+- Introduced the `Dependencies.toml` version 2. Going forward, this will be automatically managed by the `build` command and you should not modify it. If you already have a `Dependencies.toml` file, it will be automatically updated to the new version.
+- Packages in the local repository need to be configured in the `Ballerina.toml` file. The format is as follows.
```

Comment:
Are we using fullstop at the end of bullet points? Seems not consistent here and the line 703 and 704